labelbox/schema/project.py

```diff
@@ -145,6 +145,28 @@ def labels(self, datasets=None, order_by=None):
         return PaginatedCollection(self.client, query_str, {id_param: self.uid},
                                    ["project", "labels"], Label)
 
+    def queued_data_rows(self):
+        """
+        This function returns all DataRows that have not yet been labeled in this Project. 
+        It is helpful for MAL if a user only wants to upload inferences to DataRows that are still in the queue.
+        This is updated after attaching a Dataset. This will be accurate even if the Project isn't set up yet.
+
+        Returns:
+            All DataRows that are currently in the queue as a PaginatedCollection
+
+        """
+        id_param = "projectId"
+        query_str = """query GetProjectQueuedDatarowsPyApi($%s: ID!){
+        project(where:{id: $%s}){
+            labelReservations(where:{status:IN_COMPLETE, reservedBy:{id_starts_with:""}} skip: %%d first: %%d)
+            {dataRow  {%s}}}}""" % (id_param, id_param,
+                                    query.results_query_part(DataRow))
+
+        return PaginatedCollection(
```

Comment:
Something that I cannot confirm for which it may be better to pull someone else in is:

is this anything that we ought to not expose to users, like will exposing this particular method be taxing on our servers or cause us headaches down the line?

Reply:
I don't think that this is a big deal but I will confirm.